### DISCUSSION TITLE
考虑将说说功能接入Memos吗

### DISCUSSION THREAD

**Discussion by Katou-Kouseki** [Qexo/Qexo#349] [Ideas]
memos发布方式较为丰富，使用起来更加便捷

**Comment by am-abudu** [comment]
何为Memos

**Reply by Katou-Kouseki** [reply]
[memos](https://github.com/usememos/memos)
算是一个可以自部署的碎片化知识卡片吧，功能UI和flomo对标
据我了解衍生项目有：
[哔哔广场](https://immmmm.com/bbs)
[Moe Memos 客户端](https://memos.moe)
[Memos-bber Chrome 扩展](https://github.com/lmm214/memos-bber)
[Memos 微信小程序](https://github.com/Rabithua/memos_wmp)
[memos-import-from-flomo](https://github.com/JakeLaoyu/memos-import-from-flomo)
[Telegram Bot](https://github.com/qazxcdswe123/telegramMemoBot)
[A shortcut for iOS](https://sharecuts.cn/shortcut/12640)

**Reply by am-abudu** [reply]
现在打算支持的是一个更为轻量级的说说模块，所以暂时不会考虑支持。目前支持的功能已经足够实现大量的功能
